src/EnergyPlus/HVACControllers.cc

```diff
@@ -289,11 +290,11 @@ namespace HVACControllers {
 		std::string const & ControllerName,
 		int & ControllerIndex,
 		bool const FirstHVACIteration,
-		int const EP_UNUSED( AirLoopNum ), // unused1208
-		int const AirLoopPass,
```

Comment:
Will no use AirLoopNum to pass AirLoopPass data instead of passing this variable as argument. Only needed for TraceController functions.